src/models/src/apps/manifest.rs

```diff
@@ -0,0 +1,126 @@
+use rsb_derive::Builder;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+use url::Url;
+
+use crate::{SlackCallbackId, SlackShortcutType};
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackAppManifestMetadata {
+    pub major_version: Option<usize>,
+    pub minor_version: Option<usize>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackAppManifestDisplayInformation {
+    pub name: String,
+    pub description: Option<String>,
+    pub background_color: Option<String>,
+    pub long_description: Option<String>,
+}
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackAppManifestSettingsEventSubscriptions {
+    pub request_url: Option<Url>,
+    pub bot_events: Option<Vec<String>>,
```

Comment:
Nit: would be great to introduce a new type instead of String here as well to avoid changing API in future.

Reply:
Is it enough? @ 7b61588
Or defining UserEventType and BotEventType separately?

Reply:
Types are never enough :D I'm happy with one